packages/react-scripts/package.json

```diff
@@ -77,7 +77,7 @@
     "react-refresh": "^0.8.3",
     "resolve": "1.18.1",
     "resolve-url-loader": "^3.1.2",
-    "sass-loader": "^10.0.5",
+    "sass-loader": "^11.1.1",
```

Comment:
@BrodaNoel, you can update sass-loader to v12.x.x. just should CRA drop support for Node.js v10 and v12

```json
    "sass-loader": "^12.1.0",
```

Reply:
Not sure if this is a good idea.
`sass-loader@12` only allows `node >= 12.13.0`. It's a huge jump (but... to be honest, I like it).

Let's just leave this whole PR open so any collaborator with proper right can apply your suggestion and then merge it, if they consider appropiated.

Reply:
CRA currently supports node `v10`.
My commit removes support to v10, and your commit removes support for v11 and even some version in v12 (< v12.13.0)